masterlist.yaml

```
@@ -31515,3 +31515,13 @@ plugins:
         util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 16
         udr: 8
+
+  - name: 'ImprovedWaterfrontHome.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/50855/' ]
```

The URL would benefit from a name:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/50855/'
        name: 'Improved Waterfront Home'
```